src/components/network-modification-tree-pane.js

```diff
@@ -87,10 +90,9 @@ export const NetworkModificationTreePane = ({
     const DownloadIframe = 'downloadIframe';
 
     const [activeNode, setActiveNode] = useState(null);
-    const [selectedNodeIdForCopy, setSelectedNodeIdForCopy] = useState(null);
-    const [copyType, setCopyType] = useState(null);
-    const selectedNodeIdForCopyRef = useRef();
-    selectedNodeIdForCopyRef.current = selectedNodeIdForCopy;
+    const [nodeForCopy, setNodeForCopy] = useState(noSelectionForCopy);
```

Comment:
Since we now have this info in the redux, do we still need this state ?

Reply:
good point!
if the state is in redux, it is useless here.